ssc/cmod_cashloan.cpp

```diff
@@ -91,30 +91,29 @@ static var_info vtab_cashloan[] = {
     { SSC_OUTPUT, SSC_NUMBER, "total_cost", "Total installed cost", "$", "", "Financial Metrics", "*", "", "" },
 		
 
-	{ SSC_OUTPUT,        SSC_ARRAY,      "cf_energy_net",      "Energy",                  "kWh",            "",                      "Cash Flow",      "*",                     "LENGTH_EQUAL=cf_length",                "" },
-	{ SSC_OUTPUT,        SSC_ARRAY,      "cf_energy_value",      "Value of electricity savings",                  "$",            "",                      "Cash Flow",      "*",                     "LENGTH_EQUAL=cf_length",                "" },
+    { SSC_OUTPUT,       SSC_ARRAY,      "cf_energy_net",            "Electricity to grid net",       "kWh", "", "Cash Flow Electricity", "*", "LENGTH_EQUAL=cf_length", "" },
+    { SSC_OUTPUT,       SSC_ARRAY,      "cf_energy_sales",          "Electricity to grid",           "kWh", "", "Cash Flow Electricity", "*", "LENGTH_EQUAL=cf_length", "" },
```

Comment:
@cpaulgilman How do you feel about these labels for behind the meter projects:

cf_energy_net: "Electricity net generation"
cf_energy_sales: "Electricity generation"

This avoids the issue @sjanzou points out here: https://github.com/NREL/SAM/pull/927#pullrequestreview-878680572 where we're putting values that go to load in a variable with a "to grid" variable.

Reply:
> cf_energy_net: "Electricity net generation"
> cf_energy_sales: "Electricity generation"

I think those are good labels for BTM.